noxfile.py

```diff
@@ -200,7 +200,7 @@ def launch_gui_test_server(session: Session) -> None:
     session.conda_install("mysqlclient")
     if os.path.exists("/tmp/tests.sqlite"):
         os.remove("/tmp/tests.sqlite")
-    session.install("-e", "./jobmon_core")
+    session.install("-e", "./jobmon_core[otlp]")
     session.install("-e", "./jobmon_client")
     session.install("-e", "./jobmon_server")
```

Comment:
session.install("-e", "./jobmon_serve[otlp]") would be better since it includes jobmon_core[otlp] and the flask and sqlalchemy integrations.

Reply:
good note